test/basic.html

```diff
@@ -194,16 +169,10 @@
 
         testEl.$.parentFiltered.active = testEl.$.child2a;
 
-        setTimeout(function() {
-          try {
-            window.dispatchEvent(new CustomEvent('resize', { bubbles: false }));
-            expect(pendingNotifications).to.be.eql(0);
-            RemoveListeners(listeners);
-            done();
-          } catch (e) {
-            done(e);
-          }
-        }, 0);
+          window.dispatchEvent(new CustomEvent('resize', { bubbles: false }));
+          expect(pendingNotifications).to.be.eql(0);
+          RemoveListeners(listeners);
+          done();
```

Comment:
Since you've removed the asynchrony from this test, you can take out the `done` (here and on line 161).